apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -729,6 +747,31 @@ class ManageStudentsTable extends Component {
               )}
             </div>
           )}
+          {noSectionCode.includes(loginType) && (
+            <div style={styles.sectionCodeBox}>
+              {i18n.sectionCodeWithColon()}
+              <strong>
+                {` ${i18n.notApplicable()} `}
+                <a onClick={() => this.showSectionCodeDialog()}>
+                  {i18n.whyWithQuestionMark()}
+                </a>
+              </strong>
+            </div>
+          )}
+          <div>
+            <BaseDialog
+              useUpdatedStyles
+              fixedWidth={800}
```

Comment:
Our smallest supported screen size is 1024x768 https://code.org/educate/it - will this expand off-screen on these small screen sizes?

Reply:
I was working off of other dialog examples I saw, but reducing it to 600px keeps the header on a single line so I’ll go with that.

Reply:
👍 - 800 likely won't cause problems. I had width and height switched in my mind when I was making that comment. 1024 is width and 768 is height. If you ever want, you can test against smaller screen sizes by using the device toolbar in the chrome dev tools. [Some instructions on a random blog post](https://ffeathers.wordpress.com/2016/04/16/set-size-of-browser-inner-window-for-screenshots-in-chrome/#:~:text=Open%20Chrome%20Developer%20Tools.,right%20next%20to%20the%20dropdown.).